curation-migration-datasync/dynamodb/curatedItemRecordModel.ts

```diff
@@ -0,0 +1,206 @@
+import { CuratedItemRecord, ScheduledSurfaceGuid } from './types';
```

Comment:
this just consolidates all the dynamodb methods into one class, including the event body transformer logic. now only the model needs to be imported, and you don't need to pass around the dynamo client